wpilibj/athena.gradle

```diff
@@ -55,6 +57,7 @@ publishing {
     publications {
         wpilibj(MavenPublication) {
             artifact jar
+            artifact wpilibJNativeLibraries
```

Comment:
I'd much prefer this be a separate artifact, not a classifier on this one. It's not wpilibj, it's wpilibj dependencies. Something like edu.wpi.first.wpilibj:nativelibraries